src/SoundManager.h

```diff
@@ -5,34 +5,119 @@
 #include <SFML/Audio/Sound.hpp>
 #include <SFML/Audio/Music.hpp>
 
+#include <unordered_map>
+
 class SoundManager
 {
 private:
+    std::unordered_map<std::string, sf::SoundBuffer> soundBuffers;
+    std::unordered_map<uint8_t, sf::Sound*> sounds;
+    uint8_t soundId = 1;
+
+    std::unordered_map<uint8_t, sf::Music*> musics;
+    uint8_t musicId = 1;
+
 protected:
 
 public:
-    void playSound(const std::string &path, bool loop = false)
+    SoundManager () {}
+
+    ~SoundManager()
```

Comment:
Interessante esses loops, nunca tinha visto sendo feito assim no for, achei bem legivel ate.